sktime/transformations/series/holiday.py

```diff
@@ -0,0 +1,310 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements holidays transformers."""
+import pandas
+
+from sktime.transformations.base import BaseTransformer
+
+__all__ = ["CountryHolidaysTransformer", "FinancialHolidaysTransformer"]
+__author__ = ["yarnabrina"]
+
+
+class CountryHolidaysTransformer(BaseTransformer):
+    """Country Holidays Transformer.
+
+    This implementation wraps over holidays [1]_ by dr-prodigy.
```

Comment:
might be worth saying here what it does in `transform`. Does it produce a df with columns? If yes, which names, and order? Which rows?